README.md

```
@@ -64,15 +64,73 @@ Stable releases are published [on the repository releases page](../../releases/)
-
-![Updater_mc9y7o4qcL.png](assets/Updater_mc9y7o4qcL.png)
-
-![Updater_EMJNlbXitT.png](assets/Updater_EMJNlbXitT.png)
+All screenshots are taken from the included demo server (see [Run the demo locally](#run-the-demo-locally)).
+
+### Notification & changelog
+
+<table>
+<tr>
+<td width="50%">
+<img src="assets/screenshots/Updater_E2ETkBrJD1.png" width="100%" /><br/>
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)